festival/src/main/java/com/hubses/festival/web/rest/CustomerResource.java

```diff
@@ -18,34 +24,88 @@
     private UserService userService;
 
     @RequestMapping("")
-    public Iterable<User> getAllCustomers() {
-        return userService.getAllCustomers().map(customers -> customers)
+    public Iterable<CustomerModelDTO> getAllCustomers() {
+        return userService.getAllCustomers().map(customers -> {
```

Comment:
As i could recall map is called on every customer, so you don't need a list, just CustomerModelDtoBuilder and a collector at the end; in current implementation i think only the last customer information would be returned